PubnativeLite/PubnativeLite/Ad Request/PNLiteAdFactory.m

```diff
@@ -53,12 +53,32 @@ - (PNLiteAdRequestModel *)createAdRequestWithZoneID:(NSString *)zoneID
     adRequestModel.requestParameters[HyBidRequestParameter.identifierOfOMSDKIntegration] = HYBID_OMSDK_IDENTIFIER;
 //    adRequestModel.requestParameters[HyBidRequestParameter.supportedAPIFrameworks] = [supportedAPIFrameworks componentsJoinedByString:@","];
     
-    NSString* privacyString =  [[HyBidUserDataManager sharedInstance] getIABUSPrivacyString];
+    if ([HyBidSettings sharedInstance].appID != NULL) {
+        [self setAppStoreAppID:adRequestModel withAppID:[HyBidSettings sharedInstance].appID];
+        
+        NSString *adIDs = [self getSKAdNetworkIDs:adRequestModel];
+        adRequestModel.requestParameters[HyBidRequestParameter.skAdNetworkAdNetworkIDs] = adIDs;
```

Comment:
Shouldn't we check if this string is empty as well to send the params? Maybe I'm missing it somewhere. It could cause the request to fail as well, no?